### PR TITLE
Fixed the blocked version string

### PR DESCRIPTION
**Pull request by yuhuitech**
Dear developer:

Hello! I am the creator of issue #357, and I have fixed the blocked version string. I would appreciate it if you could kindly revise my code and leave me some advice. Thank you so much for your precious time!

Here is the screenshot after my changes:

copy

Thanks again! Have a nice day! :

### CODE REVIEW

File: app/src/main/res/layout/activity_about.xml [hunk 1]
```diff
@@ -36,22 +38,30 @@
...
+                app:layout_constraintLeft_toRightOf="@+id/imageView3"
+                app:layout_constraintRight_toLeftOf="@+id/appVersion"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintHorizontal_bias="0.1" />
 
             <TextView
                 android:id="@+id/appVersion"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:paddingStart="15dp"
```
Comment (spuday90) [inline app/src/main/res/layout/activity_about.xml:50]: Please add some margin here

**Review by spuday90 — CHANGES_REQUESTED** [review]
Everything looks good just need some margin between app name and version text

File: app/src/main/res/layout/activity_about.xml (hunk 1 above)
Comment (yuhuitech) [reply app/src/main/res/layout/activity_about.xml:50]: OK! I will work on this right now. :)

Comment (yuhuitech) [reply app/src/main/res/layout/activity_about.xml:50]: Hello! After some tries, I just found out that maybe the padding of "10dp" would be more suitable. Like this one:

Screenshot_1630293300

So would you agree with this one if it is OK to you? :) Thanks!

**Review by spuday90 — APPROVED** [review]

**Pull request merged**